frontend/README.md

```diff
@@ -20,6 +20,19 @@ symlink `frontend/node_modules/pcx-shared-components` to `frontend/pcx-shared-co
 and both PCA and PCP codebases can use shared components simply by 
 doing `import _ from "pcx-shared-components"`. 
 
+## Running the frontends
```

Comment:
how are we enforcing lint rules? repo wide or frontend on frontend basis? how do I run linting that's currently setup/should linting be mentioned in the README?

Reply:
Right now there's lint rules defined per-repository. You can run `yarn lint` from any repo, or also run `lerna run lint` to lint in all 4 repos (3 products + shared comps) at once. I agree this is suboptimal, @eyingxuan might have some thoughts on how to improve the setup, we can also check out other monorepos to see what they do.

Reply:
I kinda hate semicolons so I'm fine with this because I can keep the pcr eslint configs from before, but if @eyingxuan has a different opinion I'd be willing to hear it

Reply:
Depends on what the concern is - if it's mostly just duplicated lint rules everywhere we can make a base configuration file and have all our projects extend it. I still prefer having a lint script in each package just because you don't have to run everything (maybe lint-staged fixes this but still) if you clearly just changed one of the repos and it also makes overriding configs for a specific package easier.